.github/workflows/amis-workflow.yml

```diff
@@ -1,8 +1,8 @@
 ---
 name: amis-workflow
 
-on:  # yamllint disable-line rule:truthy
-  workflow_dispatch:
+on: # yamllint disable-line rule:truthy
+  workflow_dispatch: null
```

Comment:
Why is this needed?

Reply:
That is a default of the auto-formatter. See nullStr at https://eemeli.org/yaml/#tostring-options . I have changed the options to suppress that "null".